client/android/app/build.gradle

```diff
@@ -149,25 +149,36 @@ android {
 }
 
 dependencies {
-    compile project(':bugsnag-react-native')
-    compile project(':react-native-device-info')
-    compile (project(':react-native-maps')){
+    implementation project(':bugsnag-react-native')
```

Comment:
do you know if these changes mean we can't use `react-native link` anymore? You could test by finding a native module and trying to link it (make sure you do it with a clean repo so you can wipe it afterwards)

Reply:
Both will work, you will just get a "deprecated end of 2018" warning. I assume react will catch up and sort them self out before then